reVX/least_cost_xmission/least_cost_xmission_cli.py

```diff
@@ -229,47 +228,47 @@ def local(ctx, cost_fpath, features_fpath, balancing_areas_fpath,
 @click.option('--split-to-geojson', '-s', is_flag=True,
               help='After merging GeoPackages, split into GeoJSON by POI name'
               '.')
+@click.option('--suppress-combined-file', is_flag=True,
+              help='Don\'t create combined layer.')
 @click.option('--out-file', '-of', default=None, type=STR,
-              help='Name for output GeoPackage file.')
+              help='Name for output GeoPackage/CSV file.')
 @click.option('--drop', '-d', default=None, type=STR, multiple=True,
               help=('Transmission feature category types to drop from '
                     'results. Options: {}'.format(", ".join(TRANS_CAT_TYPES))))
 @click.option('--out-dir', '-od', type=click.Path(exists=True),
-              default='.', show_default=True,
+              default='./out', show_default=True,
               help='Output directory for output files. Path must exist.')
 @click.option('--simplify-geo', type=FLOAT,
               show_default=True, default=None,
               help='Simplify path geometries by a value before exporting.')
 @click.argument('files', type=STR, nargs=-1)
 @click.pass_context
-def merge_output(ctx, split_to_geojson, out_file, out_dir, drop,  # noqa
-                 simplify_geo, files):
+def merge_output(ctx, split_to_geojson, suppress_combined_file, out_file,
+                 out_dir, drop, simplify_geo, files):
     """
     Merge output GeoPackage/CSV files and optionally convert to GeoJSON
     """
     log_level = "DEBUG" if ctx.obj.get('VERBOSE') else "INFO"
     init_logger('reVX', log_level=log_level)
 
     if len(files) == 0:
-        logger.info('No files passed to be merged')
+        logger.error('No files passed to be merged')
         return
 
-    if len(files) == 1:
-        files = sorted(Path(out_dir).glob(files[0]))
-
-    logger.debug('Merging {}'.format(files))
-
     if drop:
         for cat in drop:
             if cat not in TRANS_CAT_TYPES:
                 logger.info('--drop options must on or more of {}, received {}'
                             .format(TRANS_CAT_TYPES, drop))
                 return
 
-    logger.info('Loading: {}'.format(", ".join(files)))
     warnings.filterwarnings('ignore', category=RuntimeWarning)
-    df = pd.concat([gpd.read_file(f) if "gpkg" in f else pd.read_csv(f)
-                    for f in files])
+    dfs = []
+    for i, file in enumerate(files):
```

Comment:
can you change this to  `... enumerate(files, start=1):`? That way we get (1/10) .. (10/10) instead of (0/10) .. (9/10) in the logger string

Reply:
Love it, will do